crates/houston/src/lib.rs

```diff
@@ -19,13 +19,5 @@ use std::fs;
 pub fn clear() -> Result<(), HoustonProblem> {
     let profiles_dir = home::dir()?.join("profiles");
     tracing::debug!(profiles_dir = ?profiles_dir);
-    let result = fs::remove_dir_all(profiles_dir);
-    match result {
-        Ok(()) => Ok(()),
-        Err(_) => {
-            // we should not panic if a user tries to clear and has nothing to clear
-            tracing::warn!("attemped to clear configuration but there was nothing to clear!");
-            Ok(())
-        }
-    }
```

Comment:
i don't think we should panic here, but i do think it's worth returning an error and allowing the consuming application to decide what to do with it